src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -88,6 +89,14 @@ public class CommandTestUtil {
     public static final String INVALID_EMAIL_DESC = " " + PREFIX_EMAIL + "bob!yahoo"; // missing '@' symbol
     public static final String INVALID_ADDRESS_DESC = " " + PREFIX_ADDRESS; // empty string not allowed for addresses
     public static final String INVALID_TAG_DESC = " " + PREFIX_TAG + "hubby*"; // '*' not allowed in tags
+    public static final String INVALID_COMPANY_DESC = " " + PREFIX_TAG + ""; // company cannot be empty
```

Comment:
Should be PREFIX_COMPANY?

Reply:
I removed this constant as it wasn't being used